packages/kuflow-rest/openapi/scripts/fix-genearted-code.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
```

Comment:
Fix the file name "fix-generated-code.js"

Reply:
I think that is OK :P, do you have another in mind?

Reply:
no no, I saw it!!! :+1: